sys/src/libc/port/reallocarray.c

```diff
@@ -41,7 +41,7 @@ reallocarray(void *optr, size_t nmemb, size_t size)
 	 */
 	if (nmemb == 0 || size == 0) {
 		free(optr);
-		return malloc(0);
+		return malloc(1);
```

Comment:
Malloc(0) already returns a valid pointer. What was the motivation for this change?